app/components/insights-tabs.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@ember/component';
+import { DEFAULT_INSIGHTS_INTERVAL, INSIGHTS_INTERVALS } from 'travis/services/insights';
+
+export const INSIGHTS_TABS = [
+  { slug: INSIGHTS_INTERVALS.WEEK, title: INSIGHTS_INTERVALS.WEEK.capitalize() },
```

Comment:
I would suggest to capitalize titles with css `text-transform: uppercase`. In this case, `INSIGHTS_TABS` could be simplified to plain array and thus to:
```javascript
INSIGHTS_TABS = Object.values(INSIGHTS_INTERVALS)
```

Reply:
In this case I kind of wanted the text markup itself to be capitalized rather than using `text-transform`, but yeah maybe that's a little silly.

I also manually constructed the array to ensure the correct order, not realizing that object key order can now be more officially relied upon in ES6+ https://stackoverflow.com/questions/5467129/sort-javascript-object-by-key/31102605#31102605 huh TIL

Reply:
It's totally up to you, that was just an idea, that I had during the review 😄 If you feel, that current implementation works better - that's absolutely okay! We're all senior developers and don't require guidance. The code reviews are generally aim to bring another point of view and to share knowledge, that all of us have a lot of, but maybe in different areas.

Reply:
Absolutely 😄 Although, impostor syndrome being what it is, I sometimes don't _feel_ like a senior dev haha.

I think what I'm gonna end up doing in this case is `INSIGHTS_TABS = Object.values(INSIGHTS_INTERVALS).map(slug => ({ slug, title: slug.capitalize() }));`. It is nice to loop through rather than have another manually defined array, but it still keeps the capitalization within the markup.

I have found your review comments very helpful and I've learned a lot from them. 😄 Especially in terms of Ember stuff, which I didn't really use much before joining Travis CI.